CHANGELOG.md

```diff
@@ -10,15 +10,18 @@ To publish a new release run `scripts/release` from the project directory.
 ### Added
 - Release builds now perform full link-time optimization to create a smaller
   binary.  We do recommend to also `strip` the `mdcat` binary.
-- Render SVG images in [kitty] (see [GH-114]).
+- Render SVG images in [kitty] (see [GH-114])
 - Update to reqwest 0.10.
+- Process file list as input (see [GH-115][PR-115] or [GH-54][GH-54]
```

Comment:
You don't need the second `[]` part if the text of the link's the same as the reference.  IOW, you can just write `[GH-54]` and `[GH-115]`, like in the rest of the changelog :)

Reply:
It is confusing using the same number system for PRs and Issues.  I was just being explicit.

Reply:
Github uses the same number system; in any case, unless you plan to edit the entire changelog please just follow suite.